docs/orchestration/concepts/artifacts.md

```diff
@@ -0,0 +1,88 @@
+# Artifacts <Badge text="Beta"/>
+
+The [Artifacts API](/api/latest/backend/artifacts.html) enables you to publish data from task runs that is rendered natively in the Prefect UI, both Prefect Cloud and Prefect Server. 
+
+Using the Artifacts API, you can easily publish information directly to the Prefect UI. These published artifacts are linked to specific task runs and flow runs, and artifacts can render more sophisticated information than you'd typically log, such as reports, tables, charts, images, and links to external data.
+
+Currently, the Artifacts API enables you to render the following artifact types:
+
+- Links
+- Markdown strings
```

Comment:
I would say just Markdown without the "strings"  - technically, both the link and markdown are a string data type in Python

Reply:
Good call. Changing this instance. In other places I want to maintain the "string containing markdown" specificity because, technically, users can pass any string content, (including properly escaped HTML). In the specific case of markdown we do render the styling automatically.